doc/nrf/gs_assistant.rst

```diff
@@ -64,6 +64,7 @@ There are several ways you can build an application:
 * To build on the command line, use the following steps:
 
    1. With admin permissions enabled, download and install the `nRF Command Line Tools`_.
+      When installing `nRF Command Line Tools`_ on macOS, make sure you download :guilabel:`J-link for x86-version`.
```

Comment:
Sorry for being picky, but the _download_ of J-link for x86 had already happened when the user has downloaded nRF Command Line tools.  So I think it would be better to use the word _select_ instead of _download_. Maybe even continue the sentence with: «when you get to choose between M1 and x86». Then it’s more clear that it’s the x86 part that is important, and at the same time communicate between the lines that it’s not outdated documentation that hasn’t been updated with details about M1.
Adding a short explanation would also be nice, so they know why it’s necessary, and when they might be able to switch to the M1 version: «This is needed because some of our apps run using Rosetta 2 (x86 emulation), and rely on the x86 version of J-Link.»